experiment/oss_fuzz_checkout.py

```diff
@@ -220,7 +220,24 @@ def _prepare_image_cache(project: str) -> bool:
       logger.info('%s::%s is already cached, reusing existing cache.', project,
                   sanitizer)
       continue
-    # Create cached image by building using OSS-Fuzz with set variable
+
+    # Pull the cache first
+    pull_cmd = [
+        'docker', 'pull',
+        _get_project_cache_image_name(project, sanitizer)
+    ]
+    try:
+      sp.run(pull_cmd, check=True)
+      logger.info('Successfully pulled cache image for %s', project)
+    except sp.CalledProcessError:
+      logger.info('Failed pulling image for %s', project)
+
+    if is_image_cached(project, sanitizer):
+      logger.info('pulled image for %s::%s', project, sanitizer)
+      continue
```

Comment:
We will need to check if the pull cache image is usable.
AFAIK, there are some cases making the image unusable, e.g., building fuzz target with `Make`.

The simplest check is
1. Clean `/out`.
2.  Run `recompile`.
3. Check if the fuzz target binary is in `/out` now.